rbac/management/migrations/0069_auditlog_uuid.py

```diff
@@ -0,0 +1,76 @@
+# Generated by Django 4.2.23 on 2025-06-27 21:29
+
+from django.db import migrations, models
+from django.shortcuts import get_object_or_404
+from management.role.model import Role
+from management.group.model import Group
+from management.permission.model import Permission
+import uuid
+
+
+def populate_new_uuid_field(apps, schema_editor):
+    AuditLog = apps.get_model("management", "auditlog")
+    for obj in AuditLog.objects.all():
+        obj.uuid = uuid.uuid4()
+        obj.save(update_fields=["uuid"])
+
+
+def populate_resource_uuid_field(apps, schema_editor):
+    AuditLog = apps.get_model("management", "auditlog")
+    for obj in AuditLog.objects.all():
```

Comment:
I am worried about the performance here. How many audit logs records do we have now? The iterator might hang and migration might fail. Maybe we can ignore the previous records, just leave them as they are, seems not that critical? @lpichler 

BTW, seems you forgot to call obj.save(), the update won't be saved